core/src/main/java/greencity/controller/UserController.java

```diff
@@ -1270,6 +1273,48 @@ public ResponseEntity<Object> getAllAuthorities(@RequestParam String email) {
         return ResponseEntity.status(HttpStatus.OK).body(authorities);
     }
 
+    /**
+     * Controller to get an employee`s positions and all possible related
+     * authorities to these positions.
+     *
+     * @param email {@link String} - employee email.
+     * @return {@link PositionAuthoritiesDto}
+     *
+     * @author Anton Bondar.
+     */
+    @ApiOperation(value = "Get information about an employee`s positions and all possible "
+        + "related authorities to these positions.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
+    })
+    @GetMapping("/get-positions-authorities")
+    public ResponseEntity<PositionAuthoritiesDto> getPositionsAndRelatedAuthorities(@RequestParam String email) {
+        return ResponseEntity.status(HttpStatus.OK).body(positionService.getPositionsAndRelatedAuthorities(email));
+    }
+
+    /**
+     * Controller to get a list of login employee`s positions.
+     *
+     * @param email {@link String} - employee email.
+     * @return List of {@link String} - list of employee positions.
+     *
+     * @author Anton Bondar.
+     */
+    @ApiOperation(value = "Get information about login employee`s positions.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
```

Comment:
you throw NOT_FOUND here
https://github.com/ita-social-projects/GreenCityUser/pull/236/files#diff-fb2ad6279cd37f2fdc3daf97cefacf4ba3aced726318d85fdc44509f1be8c628R52

Reply:
added